.github/pr-commands.json

```diff
@@ -248,11 +248,20 @@
       "/pkg/services/alerting/**/*",
       "/public/app/features/alerting/**/*",
       "/pkg/tests/api/alerting/**/*",
-      "/pkg/tests/alertmanager/**/*"
+      "/pkg/tests/alertmanager/**/*",
+      "/public/app/features/alerting/**/*"
     ],
     "action": "updateLabel",
     "addLabel": "area/alerting"
   },
+  {
+    "type": "label",
+    "name": "area/alerting",
+    "action": "addToProject",
+    "addToProject": {
+      "url": "https://github.com/orgs/grafana/projects/52"
```

Comment:
we often get a notification that "you can't add more items to the project" – I'm not familiar enough with the GH actions to figure out if that would block the PR from being merged (I doubt that) but just something to keep in mind

Reply:
Our project is actually in the "increased items preview" at the moment, which bumps it up to 50k issues 😌 
![image](https://github.com/user-attachments/assets/bf051e07-21af-4331-af9b-fc29766e9a07)
https://github.com/orgs/community/discussions/152407